### PR TITLE
Add new Program Logs to MagicEden spells

### PR DESCRIPTION
**Pull request by augustog**
Testing here https://dune.com/queries/1956663

### CODE REVIEW

**Comment by augustog** [comment]
slim ci will timeout due to size of model. Check test of rendered code (linked in PR description).

[DO NOT MERGE] we need to bump cluster size (I will merge once approved)

**Review by soispoke — APPROVED** [review]

**Pull request merged**
